src/httpSpansAgent.test.js

```diff
@@ -84,4 +87,24 @@ describe('HttpSpansAgent', () => {
 
     //no Error throwed
   });
+
+  const testTimeout = 350; //in MS
+  test(
+    'postSpans - reject connection timout',
+    async () => {
+      const reqBody = 'abcdefg';
+      globals.TracerGlobals.setTracerInputs({ token: 't_xxx' });
+
+      jest.spyOn(axios, 'create').mockImplementationOnce(() => {
+        return {
+          post: async () => {
+            await sleep(60000);
```

Comment:
I think that the first line in this function should be `done()` (to validate that it has been called)

Reply:
We cant use done here, but I'm added that the post really called :)